plugins/woocommerce/tests/e2e-pw/envs/default-pressable/playwright.config.js

```diff
@@ -3,12 +3,13 @@ const { devices } = require( '@playwright/test' );
 
 config = {
 	...config,
-	retries: 0,
 	projects: [
 		{
 			name: 'default',
 			use: { ...devices[ 'Desktop Chrome' ] },
-			testMatch: '**basic.spec.js',
+		},
+		{
+			name: 'Core API tests',
```

Comment:
@adimoldovan, with `name: 'Core API tests'` added here, I'm running API tests against Prressable/WPCOM. Many of them are failing, but before any further investigation, I want to check with you to see if I'm on the right path.

Reply:
🤔 If your plan is to run API tests (`plugins/woocommerce/tests/api-core-tests`), that will not work. That will run all e2e tests (everything in `plugins/woocommerce/tests/e2e-pw/tests`).